src/hooks/use-chat-context.tsx

```diff
@@ -95,42 +96,33 @@ export const ChatProvider: React.FC< ChatProviderProps > = ( { children } ) => {
 	}, [] );
 
 	useEffect( () => {
-		let isCurrent = true;
 		const run = async () => {
 			const siteId = selectedSite?.id;
 			if ( ! siteId || selectedSite.isAddingSite ) {
 				return;
 			}
-			setInitialLoad( ( prev ) => ( { ...prev, [ siteId ]: true } ) );
-			try {
-				const result = await Promise.all( [ fetchPluginList( siteId ), fetchThemeList( siteId ) ] );
-				if ( isCurrent ) {
+			initialLoad.current[ siteId ] = true;
+			Promise.all( [ fetchPluginList( siteId ), fetchThemeList( siteId ) ] )
```

Comment:
We no longer use `await` here to let promises run asynchronously from the side effect.